lib/geckoboard/dataset.rb

```diff
@@ -16,6 +16,10 @@ def put(data)
       client.put_data(id, payload_formatter.format(data))
     end
 
+    def post(data, options={})
```

Comment:
@boxofrad @carlmw I prefer `append` personally – the HTTP methods are just the "cURL" abstraction; keen to make sure the client libraries use abstractions and languages that feel right in their target language.